src/collective/volto/formsupport/locales/collective.volto.formsupport.pot

```diff
@@ -97,6 +101,10 @@ msgstr ""
 msgid "Your code to validate the email address:"
 msgstr ""
 
+#: collective/volto/formsupport/scripts/send_exported_forms.py:93
+msgid "Your forms export from {url} , until the {date}"
```

Comment:
```suggestion
msgid "Your forms export from {url}, until the {date}"
```

Reply:
jus wanted to be sure ',' will not be considered as a part of link in the email interpreter